src/analysis/Environments/Environments.test.ts

```diff
@@ -126,16 +121,15 @@ describe('Environments Component', () => {
       const tableRows: HTMLElement[] = screen.getAllByRole('row').slice(1); // skip header row
       const firstRuntimeRow: HTMLElement = tableRows[0];
       const workspaceForFirstRuntimeCell = getAllByRole(firstRuntimeRow, 'cell')[1].textContent;
-      expect(workspaceForFirstRuntimeCell).toBe(`${runtime1.labels.saturnWorkspaceName} (unavailable)`);
+      expect(workspaceForFirstRuntimeCell).toBe('undefined (unavailable)');
```

Comment:
in this uncommon case, could we display something more helpful to the end user than `undefined (unavailable)`? I'm not sure what we _should_ display instead; maybe Adam has a suggestion